src/utils/time.ts

```diff
@@ -232,3 +234,9 @@ export const getDayInterval = (date: Date | number) => ({
   start: startOfDay(date),
   end: endOfDay(date),
 });
+
+export const labelDayRoomSchedule = (startUtcSeconds: number) => {
+  if (isToday(fromUnixTime(startUtcSeconds))) return "";
+  else if (isTomorrow(fromUnixTime(startUtcSeconds))) return "Tomorrow";
+  else return formatDate(startUtcSeconds);
+};
```

Comment:
What's the intent of this change?

Reply:
@sparkletown/engineering I think this may relate to another item we have on our backlog somewhere, would be good to find which one it us, and then properly link that to this PR with closing keywords/etc.

Reply:
The intent of this is to provide the day information in the event that an event shown in the room modal isn't today. It it is tomorrow, it will return `Tomorrow`, and if the event is after tomorrow, it will return the abbreviated form of the date.